batch_test.go

```diff
@@ -0,0 +1,38 @@
+package iavl
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	dbm "github.com/cosmos/cosmos-db"
+	"github.com/stretchr/testify/require"
+)
+
+func cleanupDBDir(dir, name string) {
+	err := os.RemoveAll(filepath.Join(dir, name) + ".db")
+	if err != nil {
+		panic(err)
+	}
+}
+
+func BenchmarkBatchWithFlusher(b *testing.B) {
+
+}
+
+func benchmarkBatchWithFlusher(b *testing.B, backend dbm.BackendType) {
+	name := fmt.Sprintf("test_%x", randstr(12))
+	dir := os.TempDir()
```

Comment:
testing.B should have a tempDir now in go 1.18, can we try using that? it would remove the above cleanupDir as well

Reply:
yesss